site/docs/numbers/porting/lnpChecker.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: lnpChecker
+title: Check number(s) portability
+slug: /numbers/guides/porting/lnpChecker
+description: How to check number(s) portability
+keywords:
+  - bandwidth
+  - numbers
+  - porting
+  - portin
+  - port-in
+  - port
+  - lnp
+  - portability
+  - checker
+  - check
+image: ../../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+A number or set of numbers can be checked to see if they can be ported into the Bandwidth Network.
+:::info
+As an alternative to individual porting the `/bulkPortins` API eliminates the need for the `/lnpchecker` API by sorting the list of TNs automatically into a set of port-in requests by Port Type. For more details follow the [How to Bulk Port](/docs/numbers/guides/porting/createBulkPortins).
+:::
+
+TNs can be ported in a single `/portins` request if all the following are true:
```

Comment:
Should this be in the Create Port-in section?

Reply:
I put it in both porting and lnpchecker article because lnpchecker and porting are both parts of the same flow and user may enter the flow documentation from both ends, makes sense?